src/Applications/LDAS_App/util/inputs/obs_scaling_params/get_model_and_obs_clim_stats_GEOSldas.m

```diff
@@ -0,0 +1,767 @@
+    
+    %know which exact M09 tiles are actually administering the obs 
+    %-------------------
+    tmp_lon = central_lon(ia)+tmp_shift_lon;
+    tmp_lat = central_lat(ia)+tmp_shift_lat;
+    
+    [N_tile_in_cell_ij, tile_num_in_cell_ij] = get_tile_num_in_cell_ij(...
+                    tile_coord, tile_grid);
+     
+    this_FOV = 20;
+    option   = 'FOV_in_km';
+    %overwrite ia with actual administering tile number            
+    [ia] = get_tile_num_for_obs(tile_coord, tile_grid,...
+                N_tile_in_cell_ij, tile_num_in_cell_ij,...
+                option, this_FOV, tmp_lat, tmp_lon);
+    
+    ia = ia(ia>0 & ~isnan(ia));
+    
+    obsnum     = NaN+zeros(length(ic),1);
+    obsnum(ia) = [1:length(ia)];        
+    
+    N_tile_obs = length(ia);
+    
+    %-------------------
+    
+    if store_all_M09inM36
+        
+        %Not maintained/elaborated
+        tile_coord_tile_id = zeros(N_tile_obs,max_Hx_c);
+
+        disp(['centralizing obs on ',convert_grid,' grid before doing stats: max ',num2str(max_Hx_c),'tiles per obs cell'])
+
+        for i=1:N_tile_obs
+
+            tmp_ind = find(row_col_tmp(:,1) == unique_rc(i,1)  & row_col_tmp(:,2) == unique_rc(i,2));
+            tile_coord_tile_id(i,1:length(tmp_ind)) = tile_coord.tile_id(tmp_ind);
+
+        end
+        
+    else
+        
+       tile_coord_tile_id = tile_coord.tile_id(ia); 
+        
+    end
+    
+else
+    
+    N_tile_obs = N_tile;
+    ia         = 1:N_tile;
+    ic         = 1:N_tile;
+    obsnum     = 1:N_tile;        
+    
+end
+
+lon_out    = tile_coord.com_lon(ia); %NaN+zeros(N_tile,1);
+lat_out    = tile_coord.com_lat(ia); %NaN+zeros(N_tile,1);
+
+
+if hscale>0
+
+    for i=1:N_tile_obs
+
+        this_lat = lat_out(i);
+        this_lon = lon_out(i);
+
+        tmp_sq_distance =              ...
+        (central_lon - this_lon).^2 +  ...
+        (central_lat - this_lat).^2;
+
+        hscale_ind{i} = find( tmp_sq_distance <= hscale^2 );
+    end
+
+else 
+
+    hscale_ind = num2cell(ia);
+
+end
+  
+
+% initialize output statistics
+% Note: Rolf suggests to have all species as one dimension, rather than 
+%       N_pol and N_angle be specified here. Then subsample specifically
+%       when the files are written out.
+
+o_data     = NaN+zeros(N_pol,N_tile_obs,N_angle,w_days);
+m_data     = NaN+zeros(N_pol,N_tile_obs,N_angle,w_days);
+o_data2    = NaN+zeros(N_pol,N_tile_obs,N_angle,w_days);
+m_data2    = NaN+zeros(N_pol,N_tile_obs,N_angle,w_days);
+N_data     = NaN+zeros(N_pol,N_tile_obs,N_angle,w_days);
+
+data_out   = NaN+zeros(N_out_fields,N_tile_obs,N_angle);
+
+% -------------------------------------------------------------		  
+
+count = 0;
+
+for imonth = 1:length(run_months)
+    month = run_months(imonth); 
+
+for day = 1:days_in_month( 2014, month) %2014 = random non-leap year
+
+    if count < w_days
+        count = count + 1;
+    else
+        count = w_days;
+    end
+
+  for seconds_in_day = 0:dt_assim:(86400-1)
```

Comment:
Since GEOSldas v17.9.3, we have two config variables for the assimilation time stepping:  "LANDASSIM_DT" and "LANDASSIM_T0".
For our usual offline runs, incl. SMAP L4_SM, we have been using LANDASSIM_T0=000000, but for the coupled land-atm DA system (LADAS), we are moving towards LANDASSIM_T0=013000.  It will be a while before we have enough output from LADAS experiments to compute obs and model stats.  Nevertheless, it's probably best to add another parameter ("t0_assim") to this matlab script and its driver routine ("run_*.m").

Reply:
@gmao-qliu, I took care of this edit, but please double-check.

Reply:
double-checking done. @gmao-rreichle